### ISSUE TITLE
downloaders isInstalled check fails (npm 3)

### ISSUE THREAD

**Issue by marcelklehr** [rxaviers/cldr-data-npm#31]
Even if the latest version is already installed, when I install something else cldr-data's install script is re-run, all the previously downloaded files are gone and consequently cldr-data-downloader downloads everything *again*.

I'm not sure why all the files are being deleted. I suspect it's npm cleaning up or something :/

**Comment by rxaviers** [comment]
This repo uses https://github.com/rxaviers/cldr-data-downloader to do the download ([specific code](https://github.com/rxaviers/cldr-data-downloader/blob/master/index.js#L61-L66)).

Debug info is highly appreciated and I can help you with any specifics...

**Comment by marcelklehr** [comment]
Yep, State#isInstalled always returns false, because the downloaded files are gone. I checked with fs.readdir.

**Comment by rxaviers** [comment]
Oh, I see. Does this help https://docs.npmjs.com/cli/install#algorithm?

It smells like we should definitely implement #28.

**Comment by marcelklehr** [comment]
I've filed an npm issue over here: https://github.com/npm/npm/issues/11971

**Comment by marcelklehr** [comment]
I'm not sure #28 would be the right solution, as it completely undermines node's module system requiring a global install. That would make cldr-data even harder to install.
